arrow-optics/src/main/kotlin/arrow/optics/std/set.kt

```diff
@@ -17,3 +19,5 @@ fun <A, B> SetExtensions.toPSetK(): PIso<Set<A>, Set<B>, SetK<A>, SetK<B>> = PIs
  * [Iso] that defines the equality between a [Set] and a [SetK]
  */
 fun <A> SetExtensions.toSetK(): Iso<Set<A>, SetK<A>> = toPSetK()
+
+fun <A> At.Companion.setK(): At<SetK<A>, A, Boolean> = SetK.at()
```

Comment:
☝️  

Deprecated methods use internal singletons, which are not accessible from this file. Following #82 I have added the extension function to this file and used the deprecated method to ensure the behaviour does not change.

Unsure on the long term what should be done here